server/rosetta/client_online.go

```diff
@@ -85,14 +61,74 @@ func NewClient(cfg *Config) (*Client, error) {
 		v = "unknown"
 	}
 
+	txConfig := authtx.NewTxConfig(cfg.Codec, authtx.DefaultSignModes)
+
+	var supportedOperations []string
+	for _, ii := range cfg.InterfaceRegistry.ListImplementations("cosmos.base.v1beta1.Msg") {
```

Comment:
please don't hardcode the message name. Instead:
let's create `const` in `types/codec.go` and export it.

Reply:
How about `MsgProtoName`?  This is what we would get from the following call:
```
proto.MessageName((*Msg)(nil))
```